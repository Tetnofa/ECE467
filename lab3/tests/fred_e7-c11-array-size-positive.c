int arr1[0];
int arr2[1];
int arr3[-1];
