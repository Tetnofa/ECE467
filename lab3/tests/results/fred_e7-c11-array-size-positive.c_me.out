sema: 1:0 : size cannot be negative for array arr1
sema: 3:0 : size cannot be negative for array arr3
